modules/atomic-widgets/controls/types/select-control.php

```diff
@@ -20,9 +20,13 @@ public function set_options( array $options ): self {
 		return $this;
 	}
 
+	public function get_options() {
+		return $this->options;
+	}
+
 	public function get_props(): array {
 		return [
-			'options' => $this->options,
+			'options' => $this->get_options(),
 		];
 	}
```

Comment:
Why not directly call $this->options;

Reply:
@blackcolt needed the getter for tests so i changed it here as well